thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -306,4 +327,86 @@ export class AccountManager {
             return httpResponse;
         });
     }
+
+    /**
+     * Creates a game made by the current user.
+     */
+    public async createGame(): Promise<HttpResponse> {
+        const httpResponse: HttpResponse = {
+            "result": true,
+            "message": ""
+        }
+
+        return await fetch("/api/add/game", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": "Basic " + this.token,
+            },
+            credentials: "include"
+        }).then((response) => {
```

Comment:
Since we have async/await available, could we use that instead of `.then`? I think it'll make the async code easier to follow